web/src/components/CityContentHeader.tsx

```diff
@@ -85,17 +93,20 @@ const CityContentHeader = ({
       ]
 
   const kebabItems = [
-    <Link key='location' to={landingPath}>
-      <KebabActionItem text={t('changeLocation')} iconSrc={LocationOnOutlinedIcon} />
-    </Link>,
-    <LanguageSelector
-      key='language'
-      languageChangePaths={languageChangePaths}
-      isHeaderActionItem
-      languageCode={languageCode}
-      inKebabMenu
-      closeSidebar={() => setIsSidebarOpen(false)}
-    />,
+    <KebabItemDivider key='location'>
+      <Link to={landingPath}>
+        <KebabActionItem text={t('changeLocation')} iconSrc={LocationOnOutlinedIcon} />
+      </Link>
+    </KebabItemDivider>,
+    <KebabItemDivider key='language'>
+      <LanguageSelector
+        languageChangePaths={languageChangePaths}
+        isHeaderActionItem
+        languageCode={languageCode}
+        inKebabMenu
+        closeSidebar={() => setIsSidebarOpen(false)}
+      />
+    </KebabItemDivider>,
```

Comment:
🔧 I think this is the wrong place to add dividers for the kebab menu. Instead, this should be handled in `KebabMenu.tsx`. Perhaps it also makes sense to add utility methods to add dividers to a list of elements?